content/sensu-core/2.0/guides/clustering.md

```diff
@@ -0,0 +1,257 @@
+---
+title: "Clustering"
+linkTitle: "Clustering"
```

Comment:
What do you think about: "Running a Sensu Cluster"?

Reply:
Much more explicit.. I like it!